examples/armeria-http4s/src/main/scala/com/exmaple/http4s/armeria/ExampleService.scala

```diff
@@ -16,7 +16,7 @@ import org.http4s.armeria.server.ServiceRequestContexts
 import org.http4s.dsl.Http4sDsl
 import scala.concurrent.duration._
 
-class ExampleService[F[_]](implicit F: Async[F]) extends Http4sDsl[F] {
+class ExampleService[F[_]](implicit F: ConcurrentEffect[F], t: Timer[F]) extends Http4sDsl[F] {
```

Comment:
nit: for consistency? 
```suggestion
class ExampleService[F[_]](implicit F: ConcurrentEffect[F], T: Timer[F]) extends Http4sDsl[F] {
```